### PR TITLE
Feature services

### PR DESCRIPTION
**Pull request by condef5**
### Status
Ready

### Implementation Notes
* Add services restaurant
* Add test for actions
* Add librarie `isomorphic-fetch`
* Mock services restaurant

### CODE REVIEW

File: __mocks__/isomorphic-fetch.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+async function fetch(url) {
+  let data;
+  if (url === "http://localhost:4000/api/restaurants") {
```
Comment (christopher77) [inline __mocks__/isomorphic-fetch.js]: why do this url is static?

File: src/action.test.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import {
+  getRestaurantStart,
+  getRestaurantSuccess,
+  getRestaurantFail,
+  fetchRestaurants
+} from "./actions";
+
+jest.mock("isomorphic-fetch");
```
Comment (AngieCristina) [inline src/action.test.js]: a `mock` is use for an unit test?

File: __mocks__/isomorphic-fetch.js (hunk 1 above)
Comment (condef5) [reply __mocks__/isomorphic-fetch.js]: Yes, you're rigth, I will change in the next commit.

File: src/action.test.js (hunk 2 above)
Comment (condef5) [reply src/action.test.js]: Yes, I mock it for the requests to api in the server, You can read more [here](https://github.com/ableco/code-school/blob/master/modules/3/lessons/w5d2-testing-jest.md#mocking)

File: src/services/restaurant.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import fetch from "isomorphic-fetch";
```
Comment (AngieCristina) [inline src/services/restaurant.js]: So... What I could notice is that some lines of code are applied on this new file, `services/restaurant.js` Why did you decide to do this?

Comment (condef5) [reply src/services/restaurant.js]: Yes, I want to keep the actions free of logic or calls to the server, that's why all the requests to the server will be put in the services.

**Pull request merged**